vms/avm/asset_test.go

```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func TestGetAssetDescriptionC4T(t *testing.T) {
+func TestGetAssetDescriptionC4T(t *testing.T) { // TODO @evlekht do we still needed? That was removed by avax
```

Comment:
Was it really removed? I wonder why it is not in service_test.go file (what would be the logic place for it IMO)
If getAssetDescription is not deprecated, I suggest that we move the test into our camino_service_test.

Reply:
right, found avax test in service_test.go, our asset_test.go with avax license was added waaay back
I'll rename file to camino_service_test.go and fix license